include/librealsense2/rsutil.h

```diff
@@ -194,64 +35,48 @@ static void next_pixel_in_line(float curr[2], const float start[2], const float
     }
 }
 
-static bool is_pixel_in_line(const float curr[2], const float start[2], const float end[2])
+/* Helper inner function (not part of the API) */
+inline bool is_pixel_in_line(const float curr[2], const float start[2], const float end[2])
 {
     return ((end[0] >= start[0] && end[0] >= curr[0] && curr[0] >= start[0]) || (end[0] <= start[0] && end[0] <= curr[0] && curr[0] <= start[0])) &&
-           ((end[1] >= start[1] && end[1] >= curr[1] && curr[1] >= start[1]) || (end[1] <= start[1] && end[1] <= curr[1] && curr[1] <= start[1]));
+        ((end[1] >= start[1] && end[1] >= curr[1] && curr[1] >= start[1]) || (end[1] <= start[1] && end[1] <= curr[1] && curr[1] <= start[1]));
 }
 
-static void adjust_2D_point_to_boundary(float p[2], int width, int height)
+/* Helper inner function (not part of the API) */
+inline void adjust_2D_point_to_boundary(float p[2], int width, int height)
 {
     if (p[0] < 0) p[0] = 0;
     if (p[0] > width) p[0] = (float)width;
     if (p[1] < 0) p[1] = 0;
     if (p[1] > height) p[1] = (float)height;
 }
 
+
+/* Given a point in 3D space, compute the corresponding pixel coordinates in an image with no distortion or forward distortion coefficients produced by the same camera */
+void rs2_project_point_to_pixel(float pixel[2], const struct rs2_intrinsics* intrin, const float point[3]);
```

Comment:
Do we not want to add an error argument so we can communicate it out? Or at least a boolean return value?

Reply:
Existing API signature can't be modified - back-compat

Reply:
We can still add a boolean... your choice

Reply:
Meaning a boolean return

Reply:
There is no return code - it always calculates output value.